tests/foreman/api/test_subnet.py

```diff
@@ -278,6 +277,58 @@ def test_positive_subnet_parameters_override_impact_on_subnet(self):
         CaseLevel: System
         """
 
+        # Create subnet with valid parameters
+        parameters = [
```

Comment:
```suggestion
        parameter = [
```
As only one parameter is create hence more appropriate name would be `parameter` ..